shared/wallets/banner/index.tsx

```diff
@@ -41,6 +43,25 @@ const Banner = (props: Props) => (
           Please review.
         </Text>
       )}
+      {(props.offerAdvancedSendForm === AdvancedBanner.receiverBanner ||
+        props.offerAdvancedSendForm === AdvancedBanner.senderBanner) &&
+        props.onAction && (
+          <Box2 direction={'vertical'}>
+            {props.offerAdvancedSendForm === AdvancedBanner.receiverBanner &&
+              'This user accepts more assets than XLM. '}
```

Comment:
I think this would crash on Android, text string outside of a Kb.Text.

Reply:
Yup you were right but it crashed for a different reason on android (nesting a box inside a text). Fixed and now tested on Android.